xrtpy/response/temperature_response.py

```diff
@@ -20,42 +22,71 @@
 _h_eV_s = h.to(u.eV * u.s).value
 
 
-_CHIANTI_filename = (
-    Path(__file__).parent.absolute() / "data" / "XRT_emiss_model.default_CHIANTI.geny"
-)
-_CHIANTI_file = scipy.io.readsav(_CHIANTI_filename)
-_XRT_emiss_model_file = _CHIANTI_file["p0"]
-
-
-CHIANTI_file = {
-    "abundance_model": _XRT_emiss_model_file["ABUND_MODEL"][0],
-    "dens_model": _XRT_emiss_model_file["DENS_MODEL"][0],
-    "ioneq_model": _XRT_emiss_model_file["IONEQ_MODEL"][0],
-    "name": _XRT_emiss_model_file["NAME"][0],
-    "spectra": _XRT_emiss_model_file["SPEC"],
-    "spectra_units": _XRT_emiss_model_file["SPEC_UNITS"][0],
-    "temperature": _XRT_emiss_model_file["TEMP"][0],
-    "temp_units": _XRT_emiss_model_file["TEMP_UNITS"][0],
-    "tlength": _XRT_emiss_model_file["TLENGTH"][0],
-    "wlength": _XRT_emiss_model_file["WLENGTH"][0],
-    "wavelength": _XRT_emiss_model_file["WAVE"][0],
-    "wavelength_units": _XRT_emiss_model_file["WAVE_UNITS"][0],
+_abundance_model_file_path = {
+    "coronal_abundance_path": Path(__file__).parent.absolute()
+    / "data/chianti_emission_models"
+    / "XRT_emiss_model.default_CHIANTI.geny",
+    "hybrid_abundance_path": Path(__file__).parent.absolute()
+    / "data/chianti_emission_models"
+    / "XRT_emiss_model.default_CHIANTI_photospheric.geny",
+    "photospheric_abundance_path": Path(__file__).parent.absolute()
+    / "data/chianti_emission_models"
+    / "XRT_emiss_model.default_CHIANTI_hybrid.geny",
 }
 
+_abundance_model_data = {
+    "coronal": scipy.io.readsav(_abundance_model_file_path["coronal_abundance_path"])[
+        "p0"
+    ],
+    "hybrid": scipy.io.readsav(_abundance_model_file_path["hybrid_abundance_path"])[
+        "p0"
+    ],
+    "photospheric": scipy.io.readsav(
+        _abundance_model_file_path["photospheric_abundance_path"]
+    )["p0"],
+}
+
+list_of_abundance_name = ["coronal", "hybrid", "photospheric"]
+
+
+def resolve_abundance_model_type(abundance_model):
```

Comment:
```suggestion
def _resolve_abundance_model_type(abundance_model):
```
Make this a private function? Would need to change this below too.

Reply:
Change applied, thanks for the note!